filter-10-selinux.conf

```diff
@@ -2,7 +2,10 @@ filter {
   grok {
     add_tag => "selinux"
     tag_on_failure => "selinux_failure"
-    match => [ "message", ": avc:%{SPACE}%{SPACE}%{DATA:[selinux][avc]}  \{ %{WORD:[selinux][action]} \} for  pid=%{NUMBER:[selinux][pid]} comm=\"%{DATA:[selinux][command]}\" ((src=%{DATA:[selinux][source]})?|(name=\"%{DATA:[selinux][filename]}\" dev=\"%{DATA:[selinux][device]}\" ino=%{NUMBER:[selinux][inode]})?) scontext=%{DATA:[selinux][source][context]} tcontext=%{DATA:[selinux][target][context]} tclass=%{DATA:[selinux][target][class]} permissive=%{NUMBER:[selinux][permissive]}" ]
-       }
-}
-
+    id => "selinux"
+    match => [ "message", "avc:%{SPACE}%{SPACE}%{DATA:[selinux][avc]}  \{ %{WORD:[selinux][action]} \} for  pid=%{NUMBER:[selinux][pid]} comm=\"%{DATA:[selinux][command]}\" ((src=%{DATA:[selinux][source]})?|(name=\"%{DATA:[selinux][filename]}\" dev=\"%{DATA:[selinux][device]}\" ino=%{NUMBER:[selinux][inode]})?) scontext=%{DATA:[selinux][source][context]} tcontext=%{DATA:[selinux][target][context]} tclass=%{DATA:[selinux][target][class]} permissive=%{NUMBER:[selinux][permissive]}" ]
```

Comment:
You don't need multiple `%{SPACE}`. The grok pattern stands for "one or more spaces, tabs etc." (I'm 98% sure about that)

Reply:
Done